base/docs/helpdb/Base.jl

```diff
@@ -923,13 +923,24 @@ Stacktrace:
 BoundsError
 
 """
-    invoke(f, types <: Tuple, args...)
+    invoke(f, argtypes::Type, args...; kwargs...)
 
-Invoke a method for the given generic function matching the specified types, on
-the specified arguments. The arguments must be compatible with the specified types. This
-allows invoking a method other than the most specific matching method, which is useful when
-the behavior of a more general definition is explicitly needed (often as part of the
-implementation of a more specific method of the same function).
+Invoke a method for the given generic function `f` matching the specified types `argtypes` on the
+specified arguments `args` and passing the keyword arguments `kwargs`. The arguments `args` must
+conform with the specified types in `argtypes`, i.e. no conversions are insterted. This method
```

Comment:
I think "conform" is not much better than "compatible", what was wrong with my suggestion https://github.com/JuliaLang/julia/pull/23065#discussion_r131530843 ?

insterted -> inserted, or perhaps even better `"... no conversions are [done]/[performed]/[invoked]."`